powerline/segments/i3wm.py

```diff
@@ -16,12 +16,12 @@ def calcgrp(w):
 	return group
 
 
-def workspaces(pl):
+def workspaces(pl, strip=0):
 	'''Return workspace list
 
 	Highlight groups used: ``workspace``, ``w_visible``, ``w_focused``, ``w_urgent``
 	'''
 	return [{
-		'contents': w['name'],
+		'contents': w['name'][min(len(w['name']),strip):],
```

Comment:
The workspace switching is completely separate from the displaying anyway, you can bind any workspace to any key(-combination). Usually they are in mod+1 through mod+9. The number prefixes are used because i3 provides special functionality for workspace handling if you follow that naming convention, including the workspaces being returned in the "correct" order (and therefore being displayed correctly in powerline). However the numbers are often not aesthetically pleasant and there should be functionality for them to be stripped.